### PR TITLE
Create Greeting.yml

### LINKED ISSUE #631: ✨ [Feature request]: Add first interaction to greet a new contributor whenever they open a new issue or pull request
**Is your feature request related to a problem? Please describe.**
We should integrate [first interaction](https://github.com/marketplace/actions/first-interaction) with reactplay so that when new contributors will open a new issue or they are raising pr for the first time, They should be greeted

**Describe the solution you'd like**
Integrate first interaction with reactplay

**Describe alternatives you've considered**
There are various actions available in the marketplace but I will suggest `first interaction` because it is simple to integrate

**Additional context**
None

### PR DESCRIPTION
**Pull request by Engineervinay**
Added GitHub workflow file for greeting new users on their pull-request and issues.

> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Added Greeting functionality for users creating PR's and Issue's  on this repository.

Fixes #631

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
# Screenshots 

Screenshot (21)
Screenshot (22)

### CODE REVIEW

**Review by hasnainmakada-99 — DISMISSED** [review]
Looks good to me

**Comment by atapas** [comment]
@hasnainmakada-99 Does it need any modifications in our main Readme?

**Comment by hasnainmakada-99** [comment]
> @hasnainmakada-99 Does it need any modifications in our main Readme?

Not at all

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message:  |
+            Hello! Thank you for filing an issue.
+            If this is a bug report, please include relevant logs to help us debug the problem.
+        pr-message:  |
+            Hello! Thank you for your contribution 😊.
+            
+            If you are fixing a bug, please reference the issue number in the description.
+            If you are implementing a feature request, please check with the maintainers that the feature will be accepted first.
```
Comment (Sachin-chaurasiya) [inline .github/workflows/greetings.yml:22]:
```suggestion
            Hello! Thank you for your contribution 😊.
Please reference the issue number in the description if you are fixing a bug.
If you are implementing a feature request, please check with the maintainers that the feature will be accepted first.
```

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]

File: .github/workflows/greetings.yml (hunk 1 above)
Comment (Engineervinay) [reply .github/workflows/greetings.yml:22]: Thanks @Sachin-chaurasiya  sir, will make the required changes.

Comment (Engineervinay) [reply .github/workflows/greetings.yml]: Made the required changes.

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by hasnainmakada-99 — APPROVED** [review]
Looks good to go

**Review by atapas — APPROVED** [review]
LGTM

**Comment by atapas** [comment]
@all-contributors please add @Engineervinay  for Code

**Pull request merged**